scripts/init-env.sh

```diff
@@ -86,9 +85,11 @@ if $TNF_NON_OCP_CLUSTER; then
 
 	# IPv6
 	create_multus_annotation ipv6
-	if [ "$NUM" -ge 0 ]; then
-		# Remove last comma. Works on bash 3.2.4 on OS X:
-		#  https://unix.stackexchange.com/questions/144298/delete-the-last-character-of-a-string-using-string-manipulation-in-shell-script
-		export MULTUS_ANNOTATION="'[ ${MULTUS_ANNOTATION::-1} ]'"
-	fi
+
+	# Remove the last character (comma)
+	MULTUS_ANNOTATION="${MULTUS_ANNOTATION%?}"
```

Comment:
I ended up with something like this to preserve told functionality:

```
if [[ "$OSTYPE" == "darwin"* ]]; then
   export MULTUS_ANNOTATION="${MULTUS_ANNOTATION%,}"
else
   # Remove last comma. Works on bash 3.2.4 on OS X:
   #  https://unix.stackexchange.com/questions/144298/delete-the-last-character-of-a-string-using-string- manipulation-in-shell-script
   export MULTUS_ANNOTATION="'[ ${MULTUS_ANNOTATION::-1} ]'"
fi
```

Do we care about that old way or not?

Reply:
Keep the URL, it's not the obvious syntax.

Reply:
Fixed. I added back the comment including the URL. Tested the new way in Linux and OS X and seems to work well with only one line for both OSs